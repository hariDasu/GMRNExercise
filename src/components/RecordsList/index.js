export * from './RecordsFlatList';
export * from './RecordsListItem';
