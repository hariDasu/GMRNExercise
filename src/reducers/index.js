import {combineReducers} from 'redux';
import {records} from './recordsReducer';

export default combineReducers({
  records,
});
