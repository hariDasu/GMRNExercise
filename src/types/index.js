export const recordTypes = {
  RECORDS_REQUEST: 'RECORDS_REQUEST',
  RECORDS_SUCCESS: 'RECORDS__SUCCESS',
  RECORDS_FAILURE: 'RECORDS_FAILURE',
};
