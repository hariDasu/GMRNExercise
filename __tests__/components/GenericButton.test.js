import 'react-native';
import React from 'react';
import {GenericButton} from '../../src/components/';

import renderer from 'react-test-renderer';

it('GenericButton renders correctly across screens', () => {
  const tree = renderer.create(<GenericButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
