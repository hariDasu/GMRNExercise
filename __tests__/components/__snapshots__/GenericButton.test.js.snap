// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericButton renders correctly across screens 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#607D8B",
      "borderRadius": 25,
      "borderWidth": 0.5,
      "height": 50,
      "justifyContent": "center",
      "opacity": 1,
      "width": "80%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#212121",
          "fontSize": 16,
          "letterSpacing": 1.39,
          "lineHeight": 20,
        },
        Object {
          "color": "#212121",
        },
      ]
    }
  >
    Submit
  </Text>
</View>
`;
